projects/plugins/jetpack/modules/google-fonts/current/load-google-fonts.php

```diff
@@ -13,15 +13,20 @@
 function jetpack_get_google_fonts_data() {
 	$default_google_fonts_api_url        = 'https://fonts.gstatic.com';
 	$jetpack_google_fonts_collection_url = 'https://s0.wp.com/i/font-collections/jetpack-google-fonts.json';
+	$cache_key                           = 'jetpack_google_fonts_' . md5( $jetpack_google_fonts_collection_url );
+	$data                                = get_transient( $cache_key );
+	if ( $data === false ) {
+		$response = wp_remote_get( $jetpack_google_fonts_collection_url );
+		if ( is_wp_error( $response ) || wp_remote_retrieve_response_code( $response ) !== 200 ) {
+			return null;
+		}
 
-	$response = wp_remote_get( $jetpack_google_fonts_collection_url );
-	if ( is_wp_error( $response ) || wp_remote_retrieve_response_code( $response ) !== 200 ) {
-		return null;
-	}
+		$data = json_decode( wp_remote_retrieve_body( $response ), true );
+		if ( $data === null ) {
+			return null;
+		}
 
-	$data = json_decode( wp_remote_retrieve_body( $response ), true );
-	if ( $data === null ) {
-		return null;
+		set_transient( $cache_key, $data, HOUR_IN_SECONDS );
```

Comment:
Per discussion at p1698937537916569/1698846209.378979-slack-C01U2KGS2PQ
```suggestion
		set_transient( $cache_key, $data, DAY_IN_SECONDS );
```